Source/WebCore/platform/network/ResourceHandleInternal.h

```diff
@@ -107,7 +107,7 @@ class ResourceHandleInternal {
 
     bool m_defersLoading;
     bool m_shouldContentSniff;
```

Comment:
I wonder if this should get a similar treatment to the `ContentEncodingSniffingPolicy` member.

Reply:
I agree with that and I plan to do that in a follow-up.